saw-remote-api/python/saw_client/option.py

```diff
@@ -0,0 +1,36 @@
+from abc import ABCMeta, abstractmethod
+from typing import Any
+
+class SAWOption(metaclass=ABCMeta):
+    @abstractmethod
+    def __str__(self) -> str: pass
+
```

Comment:
```suggestion
  @abstractmethod
  def __repr__(self) -> str: pass
```
You should probably also implement `__repr__` for all of the classes in this file so users don't get unexpected output if they examine these things at the command line.

Reply:
Urk, good point. I'd really hate to make each subclass manually implement `__repr__`, however, as that seems tedious for what should be very simple classes. In fact, I'm going to just mark each of these as `@dataclass`, as that automatically defines `__repr__` the way I'd expect without any extra effort on the user's part:

```
# Before @dataclass
>>> LaxArithmetic()
<saw_client.option.LaxArithmetic object at 0x7f9d98055d00>
# After @dataclass
>>> LaxArithmetic()
LaxArithmetic()
```